tests/test_project.py

```diff
@@ -135,51 +126,51 @@ def test_dimension_map_and_reduce_in_dataset(cached_registry):
     assert len(fraction) == 1
     assert fraction[0] == (0.9 * 1.3)
 
+    # TODO DT: Do we need this?
```

Comment:
@lixiliu Can I delete this code block? I think it was testing pivoted column "map and reduce" which we no longer perform.